### PR TITLE
auto isPhrase if isPhrase not set

### LINKED ISSUE #109: Auto-consider isPhrase as true if tag.word has empty space
Fuzzy vs Exact seem to work the exact same way due to `isPhrase` current behavior in the `TagQuery` type

### PR DESCRIPTION
**Pull request by Akrion**
Fixes #109

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  The README has not been updated. Please update the README.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  We were able to automatically fix some formatting issues in this PR for you!
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/example-types/tagsQuery.js [hunk 1]
```diff
@@ -16,8 +16,9 @@ let wordPermutations = _.flow(
  */
 let addQuotesAndDistance = _.curry((tag, text) => {
   // Multiple words
-  if (tag.isPhrase)
+  if (tag.isPhrase || _.includes(' ', tag.word)) {
```
[the remark this answers is not in the document]
Comment (Akrion) [reply src/example-types/tagsQuery.js]: Doing that brakes all the other `addQuotesAndDistance` test cases which is why I did it that way.

[the remark this answers is not in the document]
Comment (daedalus28) [reply src/example-types/tagsQuery.js]: Yup, turns out it `tag.word` isn't the right thing. It should be `text`. Just pushed the fix for you - tests pass now.

[the remark this answers is not in the document]
Comment (daedalus28) [reply src/example-types/tagsQuery.js]: https://github.com/smartprocure/contexture-elasticsearch/pull/110/commits/8ebaa37c98a02082ab29d03e72cc5b1d6b4e771c

**Pull request merged**